modules/fedimint-wallet-tests/src/bin/wallet-module-tests.rs

```diff
@@ -385,6 +406,16 @@ async fn circular_deposit_test() -> anyhow::Result<()> {
                 .await?;
             assert_withdrawal(&send_client, &receive_client, bitcoind, fed).await?;
 
+            let fedimint_cli_version = FedimintCli::version_or_default().await;
+            if fedimint_cli_version >= *VERSION_0_8_0_ALPHA {
```

Comment:
@bradleystachurski do you think it's worth changing this to also run tests on the release branch?

Reply:
Yeah, might as well https://github.com/fedimint/fedimint/pull/7514